### PR TITLE
docs: Align the auth documentation with the latest changes

### PR DESCRIPTION
**Pull request by Rolika4**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement
- [x] Documentation

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Pull Request contain one commit. I squash my commits.

### CODE REVIEW

File: docs/operator-guide/auth/platform-auth-model.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+In KubeRocketCI, access control is implemented through various authorization methods. To control access to the resources uses single point - Keycloak using groups and roles, which integrates with Kubernetes RBAC.  Permissions for third-party tools are configured by Keycloak [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/). This document provides an overview of the access management entities in KubeRocketCI and various tools, including Kubernetes roles, Keycloak realm roles and Kubernetes custom resources. It also details their respective permissions and the tools they are applied to.
```
Comment (NikolayMarusenko) [inline docs/operator-guide/auth/platform-auth-model.md:3]: Kubernetes RBAC - need to discuss.  Keycloak [custom resources] - we do not have Keycloak custom resources.

File: docs/operator-guide/auth/configure-keycloak-oidc-eks.md [hunk 2]
```diff
@@ -32,196 +30,77 @@ Of particular importance within the Kubernetes ecosystem are the `RoleBindings`
...
-  cluster_identity_providers = {
-    keycloak = {
-      client_id                     = <keycloak_client_id>
-      identity_provider_config_name = "Keycloak"
-      issuer_url                    = "https://<keycloak_url>/auth/realms/<realm_name>"
-      groups_claim                  = "groups"
-    }
-  ...
-  ```
+This add-ons facilitates the installation of the Keycloak Operator and sets up a broker realm to manage traffic redirection between external Identity Providers (IdP) and internal clients. Additionally, it creates a shared realm that encompasses all clients, including to EKS, Sonar, Nexus, and Portal.
```
Comment (NikolayMarusenko) [inline docs/operator-guide/auth/configure-keycloak-oidc-eks.md:55]: The same. We do not install operator above...

File: versioned_docs/version-3.9.0/operator-guide/auth/platform-auth-model.md [hunk 3]
```diff
@@ -76,39 +77,40 @@ The table below shows the realm roles and the composite types they relate to.
...
 | Realm Group Name | Realm Name |
 | - | - |
 | `ArgoCDAdmins` | shared |
 | `ArgoCD-edp-users` | shared |
-| `edp-oidc-admins` | broker |
-| `edp-oidc-builders` | broker |
-| `edp-oidc-deployers` | broker |
-| `edp-oidc-developers` | broker |
-| `edp-oidc-viewers` | broker |
+| `edp-oidc-cluster-admins` | shared |
```
Comment (NikolayMarusenko) [inline versioned_docs/version-3.9.0/operator-guide/auth/platform-auth-model.md:90]: where we use/create this group?

File: docs/operator-guide/auth/platform-auth-model.md [hunk 4]
```diff
@@ -76,39 +77,40 @@ The table below shows the realm roles and the composite types they relate to.
...
 | Realm Group Name | Realm Name |
 | - | - |
 | `ArgoCDAdmins` | shared |
 | `ArgoCD-edp-users` | shared |
-| `edp-oidc-admins` | broker |
-| `edp-oidc-builders` | broker |
-| `edp-oidc-deployers` | broker |
-| `edp-oidc-developers` | broker |
-| `edp-oidc-viewers` | broker |
+| `edp-oidc-cluster-admins` | shared |
```
Comment (NikolayMarusenko) [inline docs/operator-guide/auth/platform-auth-model.md:90]: where we use/create this group?

**Review by SergK — APPROVED** [review]

**Pull request merged**